test/spec/form.spec.js

```diff
@@ -1880,6 +1880,33 @@ describe('Missing external instances', () => {
     });
 });
 
+describe('Adding tasks to the default evaluation cascade', () => {
+    const defaultEvaluationCascadeAdditions =
+        Form.prototype.evaluationCascadeAdditions;
+
+    let context;
+
+    beforeEach(() => {
+        const customFn = () => {
+            context = this;
```

Comment:
This doesn't work because:

<img width="938" alt="Screen Shot 2022-05-18 at 6 04 13 PM" src="https://user-images.githubusercontent.com/627350/169163429-f14e3a3c-9482-4964-85a5-70911d234ddb.png">

If any clever ideas come to mind, I would be much obliged @eyelidlessness. Otherwise, no worries, I'll figure it out!

Reply:
withdrawing question for now. Bunch of things wrong with that test.